src/component/cyfs-stack/src/resolver/device_manager.rs

```diff
@@ -127,6 +127,40 @@ impl DeviceInfoManagerImpl {
         }
     }
 
+    async fn verfiy_group_own_signs(
```

Comment:
The so-called "checksign" looks like it's just comparing hash values?

Reply:
Thanks, I will rename it.